pkg/proxy/nftables/proxier.go

```diff
@@ -210,7 +210,7 @@ func NewProxier(ipFamily v1.IPFamily,
 	nodePortAddressStrings []string,
 	initOnly bool,
 ) (*Proxier, error) {
-	nodePortAddresses := proxyutil.NewNodePortAddresses(ipFamily, nodePortAddressStrings)
+	nodePortAddresses := proxyutil.NewPrimaryNodePortAddresses(ipFamily, nodePortAddressStrings)
```

Comment:
Maybe just make `NewNodePortAddresses` take a `net.IP`, and nftables can pass `nodeIP` and the other backends can pass `nil`, and `NewNodePortAddresses` can do the right thing from there?